### ISSUE TITLE
Parllama wont start

### ISSUE THREAD

**Issue by jim-at-jibba** [paulrobello/parllama#33]
**Describe the bug**
When starting parllama I get he following error

```
Settings folder /Users/jamesbest/.parllama
Traceback (most recent call last):
  File "/Users/jamesbest/.local/bin/parllama", line 8, in <module>
    sys.exit(run())
             ^^^^^
  File "/Users/jamesbest/.local/pipx/venvs/parllama/lib/python3.12/site-packages/parllama/__main__.py", line 22, in run
    ParLlamaApp().run()
    ^^^^^^^^^^^^^
  File "/Users/jamesbest/.local/pipx/venvs/parllama/lib/python3.12/site-packages/parllama/app.py", line 132, in __init__
    super().__init__()
  File "/Users/jamesbest/.local/pipx/venvs/parllama/lib/python3.12/site-packages/textual/app.py", line 664, in __init__
    self.stylesheet = Stylesheet(variables=self.get_css_variables())
                                           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jamesbest/.local/pipx/venvs/parllama/lib/python3.12/site-packages/parllama/app.py", line 168, in get_css_variables
    settings.theme_name, self.dark
                         ^^^^^^^^^
AttributeError: 'ParLlamaApp' object has no attribute 'dark'
```

**To Reproduce**
1. Go to Kitty
2. type parllama
3. See error

**Comment by paulrobello** [comment]
This is caused by an upstream dependancy. 
The next version of parllama will have it fixed. 
I should have it out by tomorrow at the latest.

**Comment by jim-at-jibba** [comment]
Thank you so much!

**Comment by paulrobello** [comment]
Just released parllama v0.3.11
Please let me know if it resolves your issue

**Comment by jim-at-jibba** [comment]
All starts fine now. Thanks. I am looking forward to trying it out
